apps/nextjs-website/src/app/auth/expired-code/page.tsx

```diff
@@ -0,0 +1,38 @@
+import { Box, Button } from '@mui/material';
+import ExpiredCodeCard from '@/components/organisms/Auth/ExpiredCodeCard';
+import Link from 'next/link';
+import { useTranslations } from 'next-intl';
+
+const ExpiredCode = () => {
+  const t = useTranslations('auth');
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        width: '100vw',
+        minHeight: '616px',
+        backgroundImage: 'url(/images/hero.jpg)',
+        backgroundRepeat: 'no-repeat',
+        backgroundSize: 'cover',
+        backgroundPosition: 'bottom right',
+      }}
+    >
```

Comment:
I think here you can use component `PageBackgroundWrapper` that can be found here `apps/nextjs-website/src/components/atoms/PageBackgroundWrapper/PageBackgroundWrapper.tsx`